src/streamers.jl

```diff
@@ -80,17 +80,17 @@ function Streamers(io)
         # notice our `TKey` size is not the same as official TKey, can't use sizeof()
         skipped = position(io) - start
         compressedbytes = read(io, tkey.fNbytes - skipped)
-        cname = String(compression_header.algo)
+        cname = compression_header.algo
 
-        if cname == "ZL"
+        if cname == @SVector UInt8['Z', 'L']
```

Comment:
These comparisons look a bit weird but it is what it is ;) If we do such checks at more places in future, we should probably extract that or do some dispatch magic.

Reply:
true, and we have like 3 different such ifelse right now.

poly-Decompression is hard